obsei/postprocessor/interface_aggregator.py

```diff
@@ -0,0 +1,102 @@
+from obsei.postprocessor.base_postprocessor import (
+    BasePostprocessorConfig,
+    BasePostprocessor,
+    TextPayload,
+)
+from typing import Any, List, Optional
+
+
+class InterfaceAggregatorConfig(BasePostprocessorConfig):
+    aggregation_method = "most_common"
+
+    def __init__(self, **data):
+        super().__init__(**data)
+
+
+class InterfaceAggregator(BasePostprocessor):
+    def postprocess_input(
+        self, input_list: List[TextPayload], config: InterfaceAggregatorConfig, **kwargs
+    ) -> List[TextPayload]:
+        prev_document_id: int = -1
+        document = []
+        aggregated_payloads = []
+        for output in input_list:
+            if prev_document_id == -1:
+                prev_document_id = int(output.meta["document_id"])
+                document.append(output)
+
+            elif prev_document_id == int(output.meta["document_id"]):
+                document.append(output)
+            else:
+                aggregated_payloads.append(document)
+                document = [output]
+                prev_document_id = int(output.meta["document_id"])
+
+        if document and not aggregated_payloads:
+            aggregated_payloads.append(document)
+
+        return [
+            self._process_scores(config.aggregation_method, doc)
+            for doc in aggregated_payloads
+        ]
+
+    def _process_scores(
+        self, strategy: str, documents: List[TextPayload]
+    ) -> TextPayload:
+        if strategy == "most_common":
+            return self._most_common(documents)
+        if strategy == "weighted_average":
+            return self._weighted_average(documents)
+        return TextPayload(processed_text="")
+
+    def _most_common(self, documents: List[TextPayload]) -> TextPayload:
```

Comment:
It is max of score of all the chunks? in this case name `_most_common` bit miss leading. Shouldn't we rename it to something else like `max_score` or else?

Reply:
It isn't the maximum score of all chunks, it is the label that appeared the most number of times among n splits. Hence the name `most_common` . 
Should I keep this `most_common` strategy or add a `max_score` strat that chooses the label with the max score across n splits.